source/lang/golang/rabbitgo.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
`package` cannot be used `main`. It should be something else (ideally, `rabbit`) so that other codes can call this as a reference.